sdk/synapse/Microsoft.Azure.Management.Synapse/src/Microsoft.Azure.Management.Synapse.csproj

```diff
@@ -7,14 +7,12 @@
 		<PackageId>Microsoft.Azure.Management.Synapse</PackageId>
 		<Description>Azure Synapse Management SDK Library</Description>
 		<AssemblyName>Microsoft.Azure.Management.Synapse</AssemblyName>
-		<Version>0.1.0-preview.2</Version>
+		<Version>0.1.0-preview.3</Version>
```

Comment:
I should have pointed this out earlier, but our [versioning guide](https://github.com/Azure/azure-sdk-for-net/blob/master/doc/dev/Versioning.md) suggests the first version should be `1.0.0-preview.1`.
Since Synapse hasn't GAed I believe it's still ok to make the change.

Reply:
Thanks for pointing it out. I changed the version to `1.0.0-preview.1`.